src/psyclone/tests/dynamo0p3_lma_test.py

```diff
@@ -169,8 +169,17 @@ def test_fs_descriptor_wrong_type():
     name = "testkern_qr_type"
     with pytest.raises(ParseError) as excinfo:
         _ = DynKernMetadata(ast, name=name)
-    assert "each meta_func entry must be of type 'func_type'" in \
-        str(excinfo.value)
+    assert ("'meta_funcs' meta-data must consist of an array of structure "
+            "constructors, all of type 'func_type'" in str(excinfo.value))
+    # Check that the DynFuncDescriptor03 rejects it too
+    from psyclone.dynamo0p3 import DynFuncDescriptor03
+    # Create a class that just has a name property (which is not "func_type")
+    class fake_cls(object):
```

Comment:
`pycodestyle` says: `E306 expected 1 blank line before a nested definition, found 0`

Reply:
`pylint` says:
- `C0103: Class name "fake_cls" doesn't conform to PascalCase naming style (invalid-name)`
- `C0115: Missing class docstring (missing-class-docstring)`
- `R0205: Class 'fake_cls' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)`
- `R0903: Too few public methods (0/2) (too-few-public-methods)`

Reply:
Added docstring, renamed and checked with pycodestyle 2.5.0 - all clean.